jekyll/__glossary/envvars.md

```diff
@@ -0,0 +1,5 @@
+---
+term: Environment Variables
+---
+
+Environment variables store customer data that is used to set variables for a project. Only project Administrators and Owners are allowed to view or modify Environment Variables. The values are transmitted using secure protocols such as HTTPS and SSH, encrypted at rest and hashed/salted in a manner that prevents CircleCI employees from viewing them.
```

Comment:
I don't think environment variables are only used to set variables.

Change to "Environment variables store customer data used by a project."

2nd sentence: why do we capitalize Administrators, Owners, and Environment Variables? Would lowercase all of those OR link to other terms.

3rd sentence: remove "such as" and "in a manner" --> "The values are transmitted using secure protocols like HTTPS and SSH, encrypted at rest and hashed/salted to prevent CircleCI employees from viewing them."

Reply:
thanks, fixed.